frontend/src/components/adjudications/AdjudicationSection.jsx

```diff
@@ -40,15 +43,34 @@ export default function AdjudicationsSection(props) {
         setLoading(false);
       });
 
+    getNextUnjudgedPerformance(eventId, 2) //hardcoded tablet id for now
+      .then(({data}) => {
+        setNextPerformance(data)
+    });    
+
   }, []); //added the empty array so that it will only be called after the component mounts
 
   const updateAdjudications = (data) => {
     const newAdjudications = Object.assign({}, adjudications, {[data.id]: data});
     setAdjudications(newAdjudications);
   }
 
+  const goToNextPerformance = () => {
+    if(nextPerformance==null) { //catch if they have adjudicated everything
+      return
+    }
+    history.push(`/events/${eventId}/adjudications/performance/${nextPerformance.id}`)
+  }
+
   return (
-    <Section headings={headings} loading={loading} showContent={showPerformances} type="adjudication">
+
+    <Section headings={headings} loading={loading} 
+    renderNewButton={
+    <Button variant="outlined" color="primary" onClick={goToNextPerformance}>
+      Adjudicate Next
```

Comment:
please indent this section for readability - the final bracket `>` should align with `<Section`